.travis.yml

```diff
@@ -40,7 +42,9 @@ before_script:
   - composer update --prefer-source --no-interaction $COMPOSER_FLAGS
   - sudo pip install -r Resources/doc/requirements.txt
 
-script: make test
+script:
+ - make test
+ - if [ "$CS_FIXER" = "run" ]; then make cs_dry_run ; fi;
```

Comment:
I think this need to be done in the before_script before composer update is started. No need to start the test if the coding style is wrong.